content/tutorial-static_equilibrium.md

```diff
@@ -0,0 +1,389 @@
+
+d3.set_xlim(-1,1)
+d3.set_ylim(-1,1)
+d3.set_zlim(-1,1)
+
+x, y, z = np.array([0,0,0]) # defining the point of application.  Make it the origin
+
+u, v, w = forceA # breaking the force vector into individual components
+d3.quiver(x,y,z,u,v,w,color='r') # plotting forceA
+
+u, v, w = forceB
+d3.quiver(x,y,z,u,v,w,color='b') # plotting forceB
+
+
+plt.show()
+```
+
+There are two forces emanating from a single point.
+
+
+In order to simplify this problem, you can add them together to find the sum of forces.
+
+```{code-cell} ipython3
+forceC = forceA + forceB
+print('Force C =', forceC)
+
+Force C now acts as a single force that represents both A and B.
+You can plot it to see the result.
+
+```{code-cell} ipython3
+fig = plt.figure()
+
+d3 = fig.gca(projection='3d')
+
+d3.set_xlim(-1,1)
+d3.set_ylim(-1,1)
+d3.set_zlim(-1,1)
+
+x, y, z = np.array([0,0,0])
+
+u, v, w = forceA
+d3.quiver(x,y,z,u,v,w,color='r')
+u, v, w = forceB
+d3.quiver(x,y,z,u,v,w,color='b')
+u, v, w = forceC
+d3.quiver(x,y,z,u,v,w,color='g')
+
+plt.show()
+However, the goal is equilibrium.
+This means that you want your sum of forces to be $(0, 0, 0)$ or else your object will experience acceleration.
+Therefore, there needs to be another force that counteracts the prior ones.
+
+You can write this problem as $A+B+R=0$, with $R$ being the reaction force that solves the problem.
+
+In this example this would mean:
+
+$$(1, 0, 0) + (0, 1, 0) + (R_x, R_y, R_z) = (0, 0, 0)$$
+
+Broken into $x$, $y$, and $z$ components this gives you:
+
+$$\begin{cases}
+1+0+R_x=0\\
+0+1+R_y=0\\
+0+0+R_z=0
+\end{cases}$$
+
+solving for R_x, R_y, and R_z gives you a vector $R$ of $(-1, -1, 0)$.
+
+If plotted, the forces seen in prior examples should be nullified.
+Only if there is no force remaining is the system considered to be in equilibrium.
+
+```{code-cell} ipython3
+R = np.array([-1, -1, 0])
+
+fig = plt.figure()
+
+d3.set_xlim(-1, 1)
+d3.set_ylim(-1, 1)
+d3.set_zlim(-1, 1)
+
+d3 = fig.gca(projection='3d')
+
+x, y, z = np.array([0, 0, 0])
+
+u, v, w = forceA + forceB + R # add them all together for sum of forces
+d3.quiver(x, y, z, u, v, w)
+
+plt.show()
+```
+
+The empty graph signifies that there are no outlying forces. This denotes a system in equilibrium.
+
+
+# Solving Equilibrium as a sum of moments
+
+Next let's move to a more complicated application.
+When forces are not all applied at the same point, moments are created.
+
+Similar to forces, these moments must all sum to zero, otherwise rotational acceleration will be experienced.  Similar to the sum of forces, this creates a linear equation for each of the three coordinate directions in space.
+
+A simple example of this would be from a force applied to a stationary pole secured in the ground.
+The pole does not move, so it must apply a reaction force.
+The pole also does not rotate, so it must also be creating a reaction moment.
+Solve for both.
+
+Lets say a 5N force is applied 2m above the base of the pole.

```

Comment:
Is this force being applied perpendicularly? (that's what I understand from the next example, but it would be nice to make it clearer here)